src/main/sass/_page-navigation.scss

```diff
@@ -0,0 +1,105 @@
+.previous-next-navigation {
+    position: relative;
+    overflow: hidden;
+    margin-left: 0px;
+    width: 100%;
+    padding-left: 0;
+}
+
+@media only screen and  (max-width: 64em) {
+    .previous-next-navigation {
+        width: 90%;
+    }
+
+    p.download {
+      text-align: left;
+    }
+}
+
+.previous-next-navigation li {
+    font-family: "nta", Arial, sans-serif;
+    font-size: 23px;
+    line-height: 1.11111;
+    font-weight: 400;
+    text-transform: none;
+    display: block;
+    padding: 0;
+    margin: 0;
+    width: 50%;
+    float: left;
+}
+@media (max-width: 640px) {
+    .previous-next-navigation li {
+    font-size: 18px;
+    line-height: 1.11111;
+}
+}.previous-next-navigation li.next {
+    float: right;
+    text-align: right;
+}
+.previous-next-navigation li.next a {
+    padding: 15px 35px 15px 0;
+}
+.previous-next-navigation li.next a:before {
```

Comment:
it looks like there's a lot of opportunity to take advantage of sass's nesting features here.